lms/djangoapps/courseware/courses.py

```diff
@@ -753,7 +753,7 @@ def get_course_syllabus_section(course, section_key):
 
 
 @function_trace('get_courses')
-def get_courses(user, org=None, filter_=None, permissions=None):
+def get_courses(user, org=None, filter_=None, permissions=None, **kwargs):
```

Comment:
I get that there's a want to make this flexible for future use but I'd prefer just adding an `active_only` kwarg instead of unpacking all args. This would make it clearer that 1) this is an option you can set and 2) reduce possibility of hacky query modifications in the future.

```suggestion
def get_courses(user, org=None, filter_=None, permissions=None, active_only=False):
```